utils/config/daos_server.yml

```diff
@@ -249,15 +249,14 @@
 #
 #  scm_mount: /mnt/daos/1
 #
-#  # Backend SCM device type. Either use DCPM (datacentre persistent memory)
+#  # Backend SCM device type. Either use DCPM (datacenter persistent memory)
 #  # modules configured in interleaved mode (AppDirect regions) or emulated
 #  # tmpfs in RAM.
 #  # Options are:
 #  # - "dcpm" for real SCM (preferred option), scm_size ignored
 #  # - "ram" to emulate SCM with memory, scm_list ignored
 #  # Immutable after reformat.
 #
-#  # default: dcpm
```

Comment:
Does this mean there's no default defined for some of these options? I.e. the user MUST specify them? If so maybe it would make sense to have a sane default uncommented.

Reply:
Don't want to assume that we have a configured I/O server, this is a blank configuration file that is used as a zero config to enable discovery mode as soon as the service is started, when an I/O server configuration is specified in the file then it needs to have {scm/bdev}_class specified in the per server config section.

Reply:
Got it. Thanks.